content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -171,55 +171,116 @@ docker run -d --name dd-agent \
 
 {{% tab "Kubernetes" %}}
 
-You can enable APM by installing the Agent with the Datadog Helm chart. This deploys the Datadog Agent across all nodes in your Linux-based Kubernetes cluster with a DaemonSet.
+You can enable APM by installing the Agent with either:
 
-**Note**: Single step instrumentation doesn't instrument applications in the namespace where you install the Datadog Agent. It's recommended to install the Agent in a separate namespace in your cluster where you don't run your applications.
+- [Datadog Operator](#installing-with-datadog-operator)
+- [Datadog Helm chart](#installing-with-helm)
+
+<div class="alert alert-info">Single step instrumentation doesn't instrument applications in the namespace where you install the Datadog Agent. It's recommended to install the Agent in a separate namespace in your cluster where you don't run your applications.</div>
 
 ### Requirements
 
-- Make sure you have [Helm][13] installed.
+- Kubernetes v1.20+
+- [`Helm`][1] for deploying the Datadog Operator.
+- [`Kubectl` CLI][2] for installing the Datadog Agent.
+
+{{< collapse-content title="Installing with Datadog Operator" level="h4" >}}
 
-### Installation
+To enable single step instrumentation with the Datadog Operator:
```

Comment:
Let's add more context here. Following the steps below will add support of SSI in the whole cluster: we will be emitting traces for every app written in one of SSI-supported languages in the cluster.

Reply:
Also, not sure where is the best place to add, but worth mentioning that SSI is supported by Operator 1.5.0+.

Reply:
Sounds good. I'm adding that extra context to each section, and I'm adding the Operator version to step 1.